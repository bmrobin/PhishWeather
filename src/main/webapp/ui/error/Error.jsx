import React from 'react';
import './Error.css';

export function Error(props) {
  return (
    <span id='error-id' className="error">{props.error}</span>
  );
}
