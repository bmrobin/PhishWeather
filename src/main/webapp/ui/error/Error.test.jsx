import React from 'react';
import renderer from 'react-test-renderer';
import { Error } from './Error';

describe("Error component", () => {

  test("should render the error component", () => {
    let error = "an error occurred!";
    let component = renderer.create( <Error error={error} /> );
    expect(component.toJSON()).toMatchSnapshot();
  });

});
