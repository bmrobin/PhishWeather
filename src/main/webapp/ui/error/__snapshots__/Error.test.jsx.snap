// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render the error component 1`] = `
<span
  className="error"
  id="error-id"
>
  an error occurred!
</span>
`;
